### DISCUSSION TITLE
Allow selection of multiple TGs in Search Call

### DISCUSSION THREAD

**Discussion by robertlynch3** [chuot/rdio-scanner#306] [Ideas]
Rdio-Scanner is an awesome application for live playback. Going back in time to listen to older calls is hard when multiple talkgroups are used. I understand that using groups would be the current way to go back and listen to multiple talkgroups, but sometimes that is not practical as depending on departments different talkgroups are used at various situations.

The ability to select multiple talkgroups to look at in the Search Call would greatly enhance the experience. I'd love some others thoughts on this.

**Comment by chuot** [comment]
A new feature in the upcoming v7 is the ability to assign talkgroups to many groups. Then, using groups as departments, you will be able to search as expected.
